src/test/java/com/napier/sem/DAO_IntegrationTests.java

```diff
@@ -231,4 +176,107 @@ void topNCountriesIn_allAreasMatchFilter()
         // then
         for (Country country: countries) assertEquals("Europe", country.getContinent());
     }
-}
\ No newline at end of file
+
+    /**
+     * Integration tests covering the DAO.allCitiesIn method
+     */
+    // Tests that all countries in query are in the given area
+    @Test
+    void allCitiesIn_allAreasMatchFilter()
+    {
+        // given
+        String areaFilter = "Caribbean";
+
+        // when
+        ArrayList<City> cities = dao.allCitiesIn(App.REGION, areaFilter);
+
+        // then
+        for (City city: cities) assertEquals(areaFilter, city.getCountry());
```

Comment:
this test will fail as `"Caribbean"` is not a country, and that's what the assertion is checking

Reply:
Just noticed this as well

Reply:
Should be fixed